### PR TITLE
[Sekoia] Connector do not re-run and fail silently

### PR DESCRIPTION
**Pull request by helene-nguyen**
### Proposed changes

* Add print error on unknow exception (before that it was stopping with no error cause)
* Move work id in try block

### Related issues

* https://github.com/OpenCTI-Platform/connectors/issues/1958

### Related PR
* https://github.com/OpenCTI-Platform/client-python/pull/591

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality using different use cases
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: external-import/sekoia/src/sekoia.py [hunk 1]
```diff
@@ -462,6 +462,7 @@ def _add_files_to_items(self, items: List[Dict]):
     try:
         sekoiaConnector = Sekoia()
         sekoiaConnector.run()
-    except Exception:
+    except Exception as err:
+        print(err)
```
Comment (sbocahu) [inline external-import/sekoia/src/sekoia.py:466]: why not using the logger ?

File: external-import/sekoia/src/sekoia.py [hunk 2]
```diff
@@ -78,10 +78,10 @@ def run(self):
             friendly_name = "SEKOIA run @ " + datetime.utcnow().strftime(
                 "%Y-%m-%d %H:%M:%S"
             )
-            work_id = self.helper.api.work.initiate_work(
-                self.helper.connect_id, friendly_name
-            )
             try:
+                work_id = self.helper.api.work.initiate_work(
```
Comment (sbocahu) [inline external-import/sekoia/src/sekoia.py:82]: good catch 👍

File: external-import/sekoia/src/sekoia.py (hunk 1 above)
Comment (helene-nguyen) [reply external-import/sekoia/src/sekoia.py:466]: The logger cannot be used here, but if needed we can return a shorter message :)
This `print(err)` is used in most connectors, but we can improve the return message!

**Review by Megafredo — APPROVED** [review]
It's ok for me :)

**Review by aHenryJard — APPROVED** [review]
Tested locally with [Client PR](https://github.com/OpenCTI-Platform/client-python/pull/591) all good to me.

**Pull request merged**